napari/layers/utils/layer_utils.py

```diff
@@ -201,7 +210,7 @@ def calc_data_range(data, rgb=False):
 
     Returns
     -------
-    values : list of float
+    values : tuple[float, float]
```

Comment:
Nit: there's generally a preference for more informal descriptions of types (over repeats of the actual type annotation) in [the numpydoc style](https://github.com/napari/napari/blob/8c307022cc557692409f5e8bc031f1dcde4c374a/napari/utils/colormaps/colormap_utils.py#L25) and in napari - though it's a mixed bag right now...

```suggestion
    values : 2-tuple of floats
```

Reply:
Or `pair` is would be fine/better with me.